test/stresstest/broker.py

```diff
@@ -0,0 +1,18 @@
+import subprocess
+import config
+
+class Broker:
+
+    def start(self):
+        port = self.cfg.get("broker_port")
+        address = self.cfg.get("ip")
+        subprocess.run('cargo run --release --bin databroker -- --metadata vss_release_3.0.json --port %s --address %s' % (port, address), shell = True)
```

Comment:
Do we really want to have code in here to start databroker? Maybe we want. Generally speaking a Readme how all tis is intended to be used might help....

Reply:
I did this for the stresstest, that it could be started fully automated. Maybe also by a git action :) later

Reply:
I agree on that a small readme would be great, explaining the relationships between the *.py-files in this folder and how to start the tests? Just call `run_stresstest.sh`? 

Would also be good to write a line on the included *.json file. Is it just a copy of the already existing [VSS](https://github.com/boschglobal/kuksa.val/blob/feature/stresstest/data/vss-core/vss_release_3.0.json) file? If so, why is it duplicated. If just for convenience, maybe mention that it normally shall correspond to the latest version in the vss-core directory.

Reply:
The VSS file is just a copy. Was for convenience on my side - will change it. README's are always great :)